README.md

```diff
@@ -95,6 +95,10 @@ System (DNS)**.
 names (and other internet resources). The directory stores already registered
 domain names along with their corresponding IP addresses.
 
+**Hexadecimal** - Uses 16 different symbols. "0"-"9" to represent the values
+0 to 9 and "A"-"F" ("a"-"f" is acceptable too) to represent the values 10 to 15. It's common to use
+hexadecimals when working in CSS to assign colors to your elements (ex: #000000 is black).
```

Comment:
It could be worth adding an example color that isn't black e.g: `#4bc0a9` (_the teal of the @dwyl logo_) and explaining how this breaks down. 
See: https://stackoverflow.com/a/22239907/1148249 
Also, I'm a fan of _directly_ linking to the Wikipedia article for people who are curious and want to know more e.g: https://en.wikipedia.org/wiki/Hexadecimal

Reply:
No problem, I'll edit this soon. 👍

Reply:
Thanks! (_your definition is **good** BTW!_)

Reply:
I added an example with the dwyl logo color.